tgui/packages/tgui/interfaces/PlayerStatistics.tsx

```diff
@@ -0,0 +1,59 @@
+import { Section, LabeledList } from '../components';
+import { Window } from '../layouts';
+import { useBackend } from '../backend';
+
+type Data = {
+  total_clients: string;
+  living_players: string;
+  dead_players: string;
+  observers: string;
+  living_antags: string;
+};
+
+export const PlayerStatistics = () => {
+  const { data } = useBackend<Data>();
```

Comment:
you could do something like this:
```ts
const { data: { total_clients, living_players, dead_players, observers, living_antags } } = useBackend<Data>();
```
and do like `{total_clients}` instead of `{data.total_clients}`